examples/pytorch/mvgrl/README.md

```diff
@@ -0,0 +1,146 @@
+# DGL Implementation of MVGRL
```

Comment:
Is "MVGRL" the abbreviation used by others? Otherwise it should be "MVRLG".

Reply:
The authors have used it as the repo name of their own implementation. [link](https://github.com/kavehhassani/mvgrl)

Reply:
Got it.